web/client/actions/details.js

```diff
@@ -39,9 +39,9 @@ export const detailsLoaded = (mapId, detailsUri, detailsSettings) => ({
  * @memberof actions.details
  * @return {action}        type `OPEN_DETAILS_PANEL`
 */
-export const openDetailsPanel = () => ({
```

Comment:
this is not a needed change. it was only for helping debugging, so please revert